tools/@aws-cdk/cfn2ts/lib/codegen.ts

```diff
@@ -978,10 +978,11 @@ export default class CodeGenerator {
     /**
      * If '* /' occurs literally somewhere in the doc text, it will break the docstring parsing.
```

Comment:
This made me notice this comment is not accurate. 

```suggestion
     * If '*/' occurs literally somewhere in the doc text, it will break the docstring parsing.
```

Reply:
For the same reason we are trying to not have the string `*/` occur in a doc block, I cannot type this string in the explanation here (since the explanation itself occurs in a doc block).

Reply:
Doi 🤦‍♂️